packages/swarm-components/src/Button.test.jsx

```diff
@@ -42,7 +40,7 @@ describe('👀 components are visually the same', function() {
 
 		it('Primary', async () => {
 			expect(
-				await renderer.screenshot(<Button primary>Must press me</Button>)
+				await renderer.screenshot(<Button primary icon="alert">Must press me</Button>)
```

Comment:
This might make conflicts with a bunch of button tests that @jaclynj has updated in a separate PR. Just a heads up on that coordination.